src/string.cr

```diff
@@ -1,6 +1,8 @@
 require "c/stdlib"
 require "c/string"
 
+require "regex/*"
```

Comment:
Why is this included?
Should this be added explicitly in the prelude?

Reply:
if I remove this the file doesn't compile, as somehow `Regex::MatchData` isn't resolved in the `private class MatchIterator`.